js/app.js

```diff
@@ -12,4 +15,40 @@ export default {
   getSocketUrl() {
     return 'ws://localhost:8080/ws';
   },
+
+  followUnfollow(guid, type = 'follow') {
```

Comment:
I've just never seen for example a string util file called 'util/stringUtils.js' or DOM utilities called 'util/domUtils.js'.

But, if you feel strongly about it, I could totally live with it.